arch/arm/core/cortex_m/mpu/CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 zephyr_library()
 
+zephyr_library_compile_options_ifdef(CONFIG_COVERAGE_HW_TEST
```

Comment:
why not put this in one place only?

Reply:
this cant be put in one place. This is a limitation with libraries in cmake.